README.md

```diff
@@ -85,13 +85,6 @@ If the extension isn't working, we recommend following these steps to troublesho
 3. Ensure that [magicDNS and HTTPS Certificates are enabled](https://tailscale.com/kb/1153/enabling-https/) on your tailnet.
 4. If you are running `tailscaled` in a non-default path, you can set its path via the `tailscale.socketPath` setting in VS Code.
 
-## Configuration
```

Comment:
This is generated on the "Feature Contributions" page and we no longer have a maintain the list.